nx-dev/nx-dev/app/blog/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import BlogSeoComponent from './blog-seo';
+
+export default async function BlogLayout({
```

Comment:
Seems like we can just use `page.tsx` and skip `layout.tsx`? It's not reall doing much here.